app/uk/gov/hmrc/gform/views/summary/summary.scala.html

```diff
@@ -45,13 +45,7 @@
 @backLink = @{
     val href = formTemplate.formKind.fold{ _ =>
         uk.gov.hmrc.gform.gform.routes.FormController.formSection(formTemplate._id, maybeAccessCode, lastSectionNumber).path
-    } { _ =>
-        if(taskComplete) {
-            uk.gov.hmrc.gform.tasklist.routes.TaskListController.landingPage(formTemplate._id, maybeAccessCode).path
-         } else {
-            uk.gov.hmrc.gform.gform.routes.FormController.formSection(formTemplate._id, maybeAccessCode, lastSectionNumber).path
-        }
-    }
+    } { _ => uk.gov.hmrc.gform.tasklist.routes.TaskListController.landingPage(formTemplate._id, maybeAccessCode).path}
```

Comment:
Shall we remove `taskComplete` in ExtraInfoSummary. I think we don't use it anymore

Reply:
I have to  take another look at the fix because it seems like the acceptance criteria for the GFORMS-1860 ticket have regressed.

Reply:
I re-implemented it